nautobot/extras/models/change_logging.py

```diff
@@ -67,6 +67,8 @@ class ObjectChange(BaseModel):
     changed_object_type = models.ForeignKey(to=ContentType, on_delete=models.PROTECT, related_name="+")
     changed_object_id = models.UUIDField(db_index=True)
     changed_object = GenericForeignKey(ct_field="changed_object_type", fk_field="changed_object_id")
+    change_context = models.CharField(max_length=50, choices=ObjectChangeEventContextChoices, blank=True)
+    change_context_detail = models.CharField(max_length=100, blank=True)
```

Comment:
Should these be set to `editable=False` and added to `csv_headers`?

Reply:
Good thought on both. Maybe also `db_index=True` on `change_context` at least?